src/components/table/SortableTable.tsx

```diff
@@ -71,40 +59,6 @@ function SortableTable(props: SortableTableProps) {
         setOrderBy(property);
     };
 
-    const handleSelectAllClick = (
-        event: React.ChangeEvent<HTMLInputElement>
-    ) => {
-        let newSelecteds: string[] = [];
-
-        if (event.target.checked) {
-            newSelecteds = rows.map(generateRowLabelID);
-        }
-
-        setSelected(newSelecteds);
-    };
-
-    const handleClick = (_event: any, rowLabel: string) => {
-        const selectedIndex = selected.indexOf(rowLabel);
-        let newSelected: readonly string[] = [];
-
-        if (selectedIndex === -1) {
-            newSelected = newSelected.concat(selected, rowLabel);
-        } else if (selectedIndex === 0) {
-            newSelected = newSelected.concat(selected.slice(1));
-        } else if (selectedIndex === selected.length - 1) {
-            newSelected = newSelected.concat(selected.slice(0, -1));
-        } else if (selectedIndex > 0) {
-            newSelected = newSelected.concat(
-                selected.slice(0, selectedIndex),
-                selected.slice(selectedIndex + 1)
-            );
-        }
-
-        console.log('Going to set selected to', newSelected);
-
-        setSelected(newSelected);
-    };
-
```

Comment:
The table has checkbox/selector functionality that did nothing. Removing it as there were no endpoints that could use something like this.